### ISSUE TITLE
dist:zilla

### ISSUE THREAD

**Issue by msimerson** [msimerson/mail-dmarc#107]
This question is particularly directed at @marcbradshaw . How do you feel about the DZ?  This is one of the two modules I used it in and I've just entirely removed it from the other. My primary reasons for jettisoning it are:

1. having to install half of CPAN on each machine I work on
2. having the code in the github repo be different than the published module
3. it raises the bar for contributors

Seriously, I've composed and filed this issue while waiting for authordeps to install on my laptop. Yeah, sure, they were installed once but then OS updates, or perl versions, etc. and then it has to happen all over again. Slowly. Because perl, and Moose.

**Comment by msimerson** [comment]
````
$ dzil authordeps --missing | cpanm
<snip .... yawn..../>
91 distributions installed
````

**Comment by msimerson** [comment]
4. making Travis-CI testing take much longer.

**Comment by marcbradshaw** [comment]
In general, I like DZ, it does add an extra step to our internal deployment process as dz, or certain plugins for dz, break our build system in interesting ways.  The fix there is to do a dzil build on a dev box and check the result into the relevant place.

1) The extra dev dependencies are usually justified by the savings of being able to use it once installed, but I do tend to develop on a small number of systems, and I hear you over Moose, that thing takes an age to install.
2) I don't see that as a huge issue, if you are getting the code from github then generally you do (or should) know how to build it.
3) Accepted, it can raise the bar.

So, in short, I would prefer to keep it, but also understand if you really want to drop.

**Comment by msimerson** [comment]
For now, it stays. 

I don't do much perl hacking these days. When I do, my environments are all um, dated. So I have to update them.  Then I need to install a bajillion Dist::Zilla plugins, which takes forever, reminds me of my biggest gripe with perl (so..... s....l.....o.......w......!) and I impatiently wait while visions of i386, spinning hard drives, and CPAN all flash through my head. And then I notice that my POD isn't getting rewritten, but DZ didn't emit even a single peep. So I have to track down Pod::Weaver not being installed. Because DZ helpfully swallowed the error and blithely reported success. And them I'm just irritated b/c bash, sed, and I would have done the POD updates a whole lot faster.

Also, for deployments I'm in the habit of cloning git repos and running out of them. There's a 0% chance I'll be installing DZ in those cases, which means I can't just check out master, `make install`, and be off and running, make changes and test 'em real time, in a live environment, and then push back any changes to a GitHub branch when I'm done.

**Comment by rjbs** [comment]
> And then I notice that my POD isn't getting rewritten, but DZ didn't emit even a single peep.

This is not normal.  If you give me reproduction steps, I'll see what I can do about sorting it out.
